browser-interface/packages/shared/dao/sagas.ts

```diff
@@ -160,6 +176,34 @@ function* selectRealm() {
   return realm
 }
 
+function* onboardingTutorialRealm() {
+  const profile = yield select(getCurrentUserProfile)
+  if (profile) {
+    const needsTutorial = RESET_TUTORIAL || !profile.tutorialStep
+    const newTutorialFeatureFlag = yield select(getFeatureFlagVariantName, 'new_tutorial_variant')
+    const isNewTutorialDisabled =
+      newTutorialFeatureFlag === 'disabled' || newTutorialFeatureFlag === 'undefined' || HAS_INITIAL_POSITION_MARK
+    if (needsTutorial && !isNewTutorialDisabled) {
+      try {
+        const realm: string | undefined = yield select(getFeatureFlagVariantValue, 'new_tutorial_variant')
```

Comment:
Isn't this the same as `newTutorialFeatureFlag` of line 183? Avoid doing multiple consecutive select calls if possible.